schemas/groups/propulsion.json

```diff
@@ -82,6 +65,109 @@
       "units": "Pa"
     },
 
+    "alternatorVoltage": {
+      "description": "Alternator voltage",
+      "$ref": "../definitions.json#/definitions/numberValue",
+      "units": "V"
+    },
+
+    "engineTime": {
+      "description": "Total running time for engine",
+      "$ref": "../definitions.json#/definitions/numberValue",
+      "units": "s"
+    },
+
+    "engineTemperature": {
+      "description": "Engine temperature",
+      "$ref": "../definitions.json#/definitions/numberValue",
+      "units": "K"
+    },
+
+    "coolantPressure": {
+      "description": "Coolant pressure",
+      "$ref": "../definitions.json#/definitions/numberValue",
+      "units": "Pa"
+    },
+
+    "boostPressure": {
+      "description": "Engine boost (turbo, supercharger) pressure",
+      "$ref": "../definitions.json#/definitions/numberValue",
+      "units": "Pa"
+    },
+
+    "transmission": {
+      "type": "object",
+      "properties": {
+        "gear": {
+          "type": "object",
+          "properties": {
+            "value": {
+              "enum": ["Forward", "Neutral", "Reverse", "Fault"]
```

Comment:
The problem with using an enum here is that it does not allow other values to be used in a model and still have it validate. [Marine transmissions with multiple gears](http://www.coanracing.com/Catalog.asp?ProductLine=XMT) exist and it would be easy to monitor which gear was selected e.g. using GPIO inputs on a RPi running a SignalK server or on a micro-controller connected via Ethernet. 

That a _commercial_ vendor has not (yet) seen the ROI in bringing a sensor to market should not be a reason for preventing the model from representing this. Defining it as `string` with documented values would make this open.